modules/misunderstood/src/views/full/index.tsx

```diff
@@ -192,6 +247,18 @@ export default class MisunderstoodMainView extends React.Component<Props, State>
     return (
       <Container sidePanelWidth={320}>
         <SidePanel>
+          <Popover usePortal={true} position={'bottom-right'}>
+            <Button icon="calendar" className={style.filterItem}>
+              {lang.tr('module.misunderstood.dateRange')}
+            </Button>
+            <DateRangePicker
+              onChange={this.handleDateChange.bind(this)}
+              allowSingleDayRange={true}
+              shortcuts={shortcuts}
+              maxDate={new Date()}
```

Comment:
this will create a new Date object at every re-render ... I know it's quite picky but you could create a sigle date object that represents "today" and you should be good except if DateRangePicker allows a more fine grained date time than day

Reply:
i will change to "now" from dates.ts